vpd-manager/src/worker.cpp

```diff
@@ -1469,14 +1506,32 @@ std::tuple<bool, std::string>
             logging::logMessage(ex.what());
         }
 
+        // update CollectionStatus as Failure
+        types::ObjectMap l_objectMap;
```

Comment:
Move this to a utility method so that manager can reuse it.

Reply:
ok